acceptance/bundle/variables/complex/databricks.yml

```diff
@@ -51,15 +44,14 @@ targets:
     variables:
       node_type: "Standard_DS3_v3"
       cluster:
+        # complex variables are not merged, so missing variables (policy_id) are not inherited
         spark_version: "14.2.x-scala2.11"
         node_type_id: ${var.node_type}
         num_workers: 4
         spark_conf:
           spark.speculation: false
           spark.databricks.delta.retentionDurationCheck.enabled: false
-      complexvar:
-        type: complex
-        default:
-          key1: "1"
-          key2: "2"
-          key3: "3"
+    libraries:
+      - jar: "/newpath/to/jar"
+      - whl: "/newpath/to/whl"
+      - egg: "/newpath/to/egg"
```

Comment:
The nesting seems wrong. What are you defining here?

Reply:
That was it, after shifting it right it started to work.